templates/prometheus.yaml

```diff
@@ -118,6 +118,38 @@ spec:
               - source_labels: [__meta_kubernetes_namespace]
                 target_label: namespace
                 action: replace
+            - job_name: 'kubernetes-keepalived'
+              metrics_path: /snmp
+              params:
```

Comment:
I tried to add an annotation `prometheus.io/path: /snmp?target=127.0.0.1&module=keepalived` to keepalived static pod, so to merge keeplived job config with calico-node job config, but hit this bug with `%3F`: https://github.com/prometheus/prometheus/issues/5488. That's why we have to define http `params` here.